packages/brew/test_brew_functions.sh

```diff
@@ -11,7 +14,8 @@ function assertAlphanumCamelCase {
   local testValue=$1
   local expected=$2
   local actual=$(alphanumCamelCase "$testValue")
-  assert_eq "$expected" "$actual" "Alphanumeric camel case of $testValue should be equal to $expected " || TESTS_RESULT=$?
+  local msg="Alphanumeric camel case of $testValue should be equal to $expected"
+  assert_eq "$expected" "$actual" "$msg" && log_success "$MSG" || TESTS_RESULT=$?
```

Comment:
Casing is mixed up here and this file is not run during be `test_scripts.sh`. Could you rename this file `brew_functions_test.sh` to be picked up ?

Reply:
> Casing is mixed up here

I'll fix that

8c8c7ae7ebdcbacb859c67fbfbad554400496fc9

> and this file is not run during be `test_scripts.sh`. Could you rename this file `brew_functions_test.sh` to be picked up ?

I've tried that, but unfortunately the script doesn't pass on `master` either. So for now I'll leave it alone, but raise as a separate issue for the future.

Reply:
...actually this might be a Mac + `sed` issue. Let me take another look.

Reply:
8547cd09fca299714bca3d38bd97693af1d60ffa

Renaming them works on non-Mac platforms.